lib/elixir/lib/map.ex

```diff
@@ -838,7 +828,11 @@ defmodule Map do
 
   """
   @spec equal?(map, map) :: boolean
+  def equal?(map1, map2)
```

Comment:
It's not needed but I think it's nice to have it so we don't have to think about the clauses below and can move them around/change them without thinking about how it will affect the docs.